File: library/src/main/java/com/yzyfdf/library/callback/OnTextChangedListener.java
package com.yzyfdf.library.callback;

import android.text.TextWatcher;

/**
 * @author sjj , 2019/4/25 10:01
 *
 * 文字改变后监听
 */
public abstract class OnTextChangedListener implements TextWatcher {
    @Override
    public void beforeTextChanged(CharSequence s, int start, int count, int after) {

    }

    @Override
    public void onTextChanged(CharSequence s, int start, int before, int count) {

    }
}

File: library/src/main/java/com/yzyfdf/library/callback/OnTextChangedListener.kt
package com.yzyfdf.library.callback

import android.text.TextWatcher

/**
 * @author sjj , 2019/4/25 10:01
 *
 * 文字改变后监听
 */
abstract class OnTextChangedListener : TextWatcher {
    override fun beforeTextChanged(s: CharSequence, start: Int, count: Int, after: Int) {

    }

    override fun onTextChanged(s: CharSequence, start: Int, before: Int, count: Int) {

    }
}
